packages/notifi-react/lib/context/NotifiHistoryContext.tsx

```diff
@@ -303,6 +316,8 @@ export const NotifiHistoryContextProvider: FC<
         historyItems,
         unreadCount: unreadCount ?? 0,
         hasNextPage: cursorInfo.hasNextPage,
+        isIncludeRead,
```

Comment:
Do we have to expose `isIncludeRead`. For now, I did not see the necessary use case of using `isIncludeRead`, so I would un-expose this one to make the exposed context props as few as possible. 
Hence, if it become needed, we can expose when we need that. WDYT